include/cuda/std/initializer_list

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the CUDA Toolkit, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _CUDA_INITIALIZER_LIST
+#define _CUDA_INITIALIZER_LIST
+
+#include "detail/__config"
+
+#include <initializer_list>
+
+namespace cuda {
+  namespace std {
+    using ::std::initializer_list;
+  }
+}
```

Comment:
Should be like this, right? Otherwise it misses the inline namespace? 
```suggestion
_LIBCUDACXX_BEGIN_NAMESPACE_STD
    using ::std::initializer_list;
_LIBCUDACXX_END_NAMESPACE_STD

```

Reply:
Not sure!

I think it's fine to include the ABI as the namespace include will still resolve to a 'std::' thing. I'll make the switch in another patch just since it's consistent.